tensorboard/plugins/core/core_plugin.py

```diff
@@ -576,3 +576,13 @@ def _gzip(bytestring):
     with gzip.GzipFile(fileobj=out, mode="wb", compresslevel=3, mtime=0) as f:
         f.write(bytestring)
     return out.getvalue()
+
+
+def _parse_dict(value):
```

Comment:
Optional, but maybe rename to `_parse_samples_per_plugin` or similar, to
avoid giving the impression that this parses arbitrary dicts or that it
accepts standard dict notation (`"{\"key\": \"value\"}"`)?

Reply:
Done.